docs/guide/basics/examples/button-example/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import { ThumbsUp } from "lucide-react";
+
+function Button() {
```

Comment:
The file could be renamed to `LikeButton` as well.
```suggestion
function LikeButton() {
```